packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -466,7 +466,7 @@ export default class BackupNg {
 
           const localTaskIds = { __proto__: null }
           for await (const log of logsStream) {
-            result = handleBackupLog(log, {
+            result = handleBackupLog(this._app, log, {
```

Comment:
This does not work: not passing a parameter will shift the position of others.

Reply:
But this parameter is always sent to the function (even when it's not use)

Reply:
You're not passing `jobName` which change the position of the other args.